docs/content/latest/legal/cla.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Yugabyte contributor license agreement (CLA)
+headerTitle: Yugabyte contributor license agreement (CLA)
+linkTitle: Contributor license agreement (CLA)
+description: Yugabyte individual contributor license agreement (CLA)
+menu:
+  latest:
+    parent: legal
+    identifier: yugabyte-cla
+    weight: 40
+isTocNested: true
+showAsideToc: true
+---
+
+
+Before individual contributions to the YugabyteDB project are accepted, the contributor must complete and sign the online [Yugabyte Individual Contributor License Agreement (CLA)](https://docs.google.com/forms/d/11hn-vBGhOZRunclC3NKmSX1cvQVrU--r0ldDLqasRIo/viewform?edit_requested=true) form. A plain text copy of the agreement is shown below. You can also obtain a copy of the Individual Contributor License Agreement - V.07/26/2018 at:
+
+[https://docs.google.com/document/d/1Yu_uCcAAPhrKAtq2Bb_qj_NbHMigaLEhbpFuFwP1azk/edit#](https://docs.google.com/document/d/1Yu_uCcAAPhrKAtq2Bb_qj_NbHMigaLEhbpFuFwP1azk/edit#)
+
+```
+Individual Contributor License Agreement – V.07/26/2018
+
+This Individual Contributor License Agreement (“Agreement”) is entered into between YugaByte, Inc., a Delaware corporation (“YugaByte,” “we” or “us” etc.) and you (as defined below). By submitting the Individual Contributor License Agreement form below, you hereby agree to the following terms for your present and future contributions submitted to YugaByte. Except for the license granted herein to YugaByte and recipients of software distributed by YugaByte, You reserve all right, title, and interest in and to Your Contributions.
```

Comment:
change `YugaByte` to `Yugabyte` in this page

Reply:
not relevant since page is removed